src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -478,7 +478,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
   </Target>
 
   <Target Name="_GenerateComposites"
-          Condition="'$(TargetOsName)' == 'linux'"
+          Condition="'$(BuildOsName)' == 'linux'"
```

Comment:
If I'm right to understand this change is supposed to cater for publishing both <code>linux</code> and <code>linux-musl</code> containers, I don't think this is the ideal way to do that as semantically the condition is indeed tied to the targeting, not the build OS. I would suggest using either an OR clause

<code>"'$(TargetOSName)' == 'linux' or '$(TargetOSName)' == 'linux-musl'"</code>

or

<code>"$(TargetOSName).StartsWith('linux')"</code>

Reply:
Oh I changed it to `BuildOsName` as per @wtgodbe's suggestion in the first composites PR here: https://github.com/dotnet/aspnetcore/pull/42506/files#r928022560

Reply:
Hmm, OK, I don't see it as blocking, feel free to keep it, we'll see what happens in the future if we decide to further modify the set of OS / architecture combinations supporting composite images.

Reply:
https://github.com/dotnet/aspnetcore/blob/bec278eabea54f63da15e10e654bdfa4168a2479/src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj#L106

`BuildOsName` already covers this, I think it's the right change here